test-server.js

```diff
@@ -1,24 +1,27 @@
 const request = require('supertest');
 const expect = require('chai').expect;
 
-
-describe('test server', function () {
+describe('test server', () => {
     let server;
 
-    beforeEach(function () {
+    beforeEach(() => {
         server = require('./server.js');
     });
 
-    afterEach(function () {
+    afterEach(() => {
         server.close();
     });
 
     it('responds to /api/home/', (done) => {
-        request(server).get('/api/home/').expect(200, done);
+        request(server)
+            .get('/api/home/')
```

Comment:
I like the neater code, but for next time please add boyscout clean up as a separate commit just to make the review process a bit easier